code/datums/components/thermite.dm

```diff
@@ -61,4 +61,4 @@
 
 /datum/component/thermite/proc/attackby_react(obj/item/thing, mob/user, params)
 	if(thing.is_hot())
-		thermite_melt(user)
\ No newline at end of file
+		thermite_melt(user)
```

Comment:
Would you mind adding a new line at the end of the file to prevent the no newline thing in the future?